style.css

```diff
@@ -22,3 +22,68 @@ body {
 }
 
 /* Add styles here! */
+
+/* Typography */
+h1 {
+  font-size: 2rem;
+  margin-block: 32px;
+}
+
+h2 {
+  font-size: 1.5rem;
+  margin-block: 24px;
+}
+
+p {
+  font-size: 1rem;
+  line-height: 1.5;
+  margin-block: 16px;
+}
+
+/* Layout */
+header {
+  padding-block: 64px;
+}
+
+main {
+  padding-block: 64px;
+  border-top: 8px solid var(--color-gray-medium);
+  background-color: var(--color-gray-light);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**👏 praise:** ‏Good usage of the specific property and not the background shortcut